package.json

```diff
@@ -98,6 +98,9 @@
     "typechain": "^5.0.0",
     "typescript": "^4.2.2"
   },
+  "overrides": {
+    "@uniswap/sdk-core": "4.0.6"
```

Comment:
maybe dumb question, but why do we need to override this?

Reply:
https://github.com/Uniswap/smart-order-router/pull/261#discussion_r1235929311